server/sandbox_run.go

```diff
@@ -20,6 +20,7 @@ import (
 
 // RunPodSandbox creates and runs a pod-level sandbox.
 func (s *Server) RunPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest) (*pb.RunPodSandboxResponse, error) {
+	s.Update()
```

Comment:
It's pretty awful to have this check on every request but I can see it's needed. BTW, should this check the error and log it at least?